### DISCUSSION TITLE
Option to use cloud games on anyone's project

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#2285] [Ideas] [answered]
Currently, the cloud games extension only supports 15 of Griffpatch's most popular games. This doesn't seem quite fair as some people don't even know Griffpatch and would still like to use the feature for other people's projects. I think they should be able to choose the projects they want to appear on the cloud games extension by typing the project ID, and it would add it to the list. There could still be a few Griffpatch games there by default, but then they could add/remove whatever cloud projects they want. I personally want to use cloud games on RokCoder's Uno Deluxe, but I can't.

**Comment by cobaltt7** [comment, marked as answer]
https://github.com/ScratchAddons/ScratchAddons/issues/1652
